keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2025,6 +1983,161 @@ Beta:
+    - Components depending on the feature gate:
+      - kube-apiserver
+
+* **Does enabling the feature change any default behavior?** No.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?** Yes, the feature is feature-gated.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  The feature will be restored.
+
+* **Are there any tests for feature enablement/disablement?**
+  Yes, but very limited testing of feature enablement/disablement exists.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?** A
+  misconfiguration could cause apiserver requests to be rejected, which could
+  have widespread impact such as: (1) locking an administrator out of their
+  system, (2) rejecting controller requests, thereby bringing a lot of things
+  to a halt, (3) dropping node heartbeats, which may result in overloading
+  other nodes, (4) cause kube-proxy requests to apiserver to fail, breaking
+  existing workloads. <!-- anything else catastrophic? :) -->
+
+* **What specific metrics should inform a rollback?** An abnormal spike in the
+  `apiserver_flowcontrol_rejected_requests_total` metric should potentially be
+  viewed as a sign that kube-apiserver is rejecting requests, potentially
+  incorrectly. The `apiserver_flowcontrol_request_queue_length_after_enqueue`
+  metric getting too close to the configured queue length could be a sign of
+  insufficient queue size (or a system overload), which can be precursor to
+  rejected requests.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  <!-- I don't have the knowledge to answer this. -->
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  <!-- Are max-requests-inflight, etc. being deprecated in favour of APF? -->
+
+### Monitoring Requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+  If the `apiserver_flowcontrol_dispatched_requests_total` metric is non-zero,
+  this feature is in use.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+the health of the service?**
+  - [x] Metrics
+    - Metric name: `apiserver_flowcontrol_request_queue_length_after_enqueue`
+    - Components exposing the metric: kube-apiserver
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  No SLOs are proposed for the above SLI.
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?** No.
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+  No.
+
+### Scalability
+
+<!-- I don't have the knowledge to answer some questions in this section. -->
+
+* **Will enabling / using this feature result in any new API calls?** No.
+  Describe them, providing:
```

Comment:
nit: let's remove this - this is supposed to make the answering the question easier